### DISCUSSION TITLE
Pushing into repo

### DISCUSSION THREAD

**Discussion by hcordash** [community/community#50755] [Repositories]
### Select Topic Area

General

### Body

Hey

Currently Im trying Ive added a few files and committed in my repo on linux, and now im trying to push all my changed into the github repo. I get prompted to put my username and password since its the first push but it says Authentication failed, support for password authentication was removed Augest 13, 2021.

Im putting in my correct password and username which Ive used to sign into github so Im not sure what to do.

Thanks

**Comment by airtower-luna** [comment]
You cannot use your password for `git push` or other Git operations (but you still need it to log in). You need to [create a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and enter that instead when Git asks for your password. When creating the token make sure to select the **repo** scope (for repository access, like pushing) and if you use Actions also **workflow** (required to modify workflow files).
